pvlib/pvsystem.py

```diff
@@ -849,6 +849,22 @@ def pvwatts_ac(self, pdc):
         return inverter.pvwatts(pdc, self.inverter_parameters['pdc0'],
                                 **kwargs)
 
+    def irradiance_loss_pvsyst(self, effective_irradiance,
```

Comment:
This method does not extract any data from a `PVSystem` attribute, so it's not clear to me that it's worthwhile.